hack/update-deps.sh

```diff
@@ -14,22 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-readonly ROOT_DIR=$(dirname $0)/..
-source ${ROOT_DIR}/vendor/knative.dev/test-infra/scripts/library.sh
+readonly ROOT_DIR=$(dirname "$0")/..
+# shellcheck disable=SC1090
```

Comment:
Do we really want editor specific comments like these disables?  Are they in any other knative repo atm?

Reply:
This isn't editor checks, but lint checks. It's a standard linter for shell scripts, that everyone should use: [shellcheck](https://www.shellcheck.net/).

Reply:
sure, but that's outside the scope of this PR, and introduces a new set of coding standards into the repo.  lets keep the PR specific to testing upgrades across versions please instead of commiting a variety of other comments and increasing the size of the end commit.

Reply:
Okay. I'll move those to separete PR.

Reply:
Done in b8bcde5a20aa034e5f6bcf3e92e2e4d39bfaade5